### ISSUE TITLE
Some questions about training

### ISSUE THREAD

**Issue by YCHang686** [ChaofWang/AWSRN#15]
Hello, I find that you use 'restart' to train the model, which means 1000 epochs plus 1000 epochs with new learning rate.
Can you tell me why you use 'restart'?
Does 'restart' has some benefits? @ChaofWang
